genesis/process/metaGenesisBlockCreator.go

```diff
@@ -252,15 +259,9 @@ func createProcessorsForMetaGenesisBlock(arg ArgsGenesisBlockCreator, enableEpoc
 		DataPool:           arg.Data.Datapool(),
 		CompiledSCPool:     arg.Data.Datapool().SmartContracts(),
 		NilCompiledSCStore: true,
+		EpochNotifier:      epochNotifier,
```

Comment:
\+ `SaveAccountsIfErrorDisableEpoch: enableEpochs.SaveAccountsIfErrorDisableEpoch`
and add in `genesis/process/shardGenesisBlockCreator.go` createGenesisConfig function, `SaveAccountsIfErrorDisableEpoch: unreachableEpoch`

Reply:
done, although not needed :D

Reply:
this is needed in hardfork.